src/main/java/com/palantir/ptoss/cinch/core/DefaultBindableModel.java

```diff
@@ -21,7 +23,13 @@
  */
 public class DefaultBindableModel implements BindableModel {
 
-    private final List<Binding> bindings = new CopyOnWriteArrayList<Binding>();
+    // must be transient because Bindings aren't serializable, and
+    // subclasses might be serialized.
+    private transient volatile List<Binding> bindings;
```

Comment:
I don't think it needs to be volatile though.

Reply:
These models should generally only be accessed on the Swing EDT and the `volatile` keyword suggests otherwise. Let's kill it unless there's a compelling reason.